<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerOptionsWidget</class>
 <widget class="QWidget" name="ColorPickerOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Picker</string>
  </property>
  <layout class="QGridLayout">
   <item row="5" column="0">
    <widget class="QTreeWidget" name="listViewChannels">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Channel</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Sample radius:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="radius">
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>900</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="cbNormaliseValues">
     <property name="text">
      <string>Show colors as percentages</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbPalette">
       <property name="text">
        <string>Add to palette:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SqueezedComboBox" name="cmbPalette"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cbUpdateCurrentColor">
     <property name="text">
      <string>Update current color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="cmbSources">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>32767</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Sample All Visible Layers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Current Layer</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SqueezedComboBox</class>
   <extends>QComboBox</extends>
   <header>squeezedcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
